airbyte-integrations/bases/base-normalization/integration_tests/resources/test_primary_key_streams/data_input/messages.txt

```diff
@@ -16,3 +16,7 @@
 
 {"type": "RECORD", "record": {"stream": "nested_stream_with_complex_columns_resulting_into_long_names", "emitted_at": 1602638599000, "data": { "id": 4.2, "date": "2020-08-29T00:00:00Z", "partition": { "double_array_data": [[ { "id": "EUR" } ]], "DATA": [ {"currency": "EUR" } ] } }}}
 {"type": "RECORD", "record": {"stream": "nested_stream_with_complex_columns_resulting_into_long_names", "emitted_at": 1602638599100, "data": { "id": "test record", "date": "2020-08-31T00:00:00Z", "partition": { "double_array_data": [[ { "id": "USD" } ], [ { "id": "GBP" } ]], "DATA": [ {"currency": "EUR" } ]} }}}
+
+{"type": "RECORD", "record": {"stream": "dedup_cdc_excluded", "emitted_at": 1602638599000, "data": { "id": 1.0, "val": "Mazda", "_ab_cdc_lsn": 2602638599000 }}}
```

Comment:
why did you declare `id` as `integer` but provides values of `1.0`?

Reply:
```
{"type":"RECORD","record":{"stream":"dedup_cdc_excluded","data":{"id":1,"name":"honda","_ab_cdc_updated_at":1624490365120,"_ab_cdc_lsn":22932208,"_ab_cdc_deleted_at":null},"emitted_at":1624490382845,"namespace":"public"}}
```
this is the new format sorry